aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/ReadHandler.java

```diff
@@ -39,6 +39,7 @@
                                     .port(targetDBCluster.port().toString()).build())
                             .readEndpoint(ReadEndpoint.builder()
                                     .address(targetDBCluster.readerEndpoint()).build())
+                            .dBClusterResourceId(targetDBCluster.dbClusterResourceId())
```

Comment:
thanks for your contribution. This is a good start but Create/Update/Read handlers should output same set of properties

Reply:
This is a read-only property, so I think that means it only needs to be in the Read handler; as far as I know there's no way to set this on Create or Update.

Reply:
> The model MUST contain all properties that have values, including any properties that have default values, and any readOnlyProperties as defined in the resource schema

[contract agreement](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-test-contract.html#resource-type-test-contract-create)

read only properties are mainly used for GetAtt or something that could not be provided via template but we require consistent output between handlers

Reply:
I think I understand, but I don't see where the existing read-only properties are being set in the CreateHandler or UpdateHandler code paths. :-( Can you point me to an example of this in practice, so I can use that as a model?